.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+Your issue may already be reported!
+Please search on the [issue tracker](../) before creating one.
```

Comment:
IMO: it's better to use the link (https://github.com/actix/actix-web/issues) directly to me.

Reply:
Updated!